pylabrobot/resources/plate.py

```diff
@@ -214,13 +214,19 @@ def disable_volume_trackers(self) -> None:
     """Disable volume tracking for all wells in the plate."""
 
     for well in self.get_all_items():
-      well.tracker.disable()
+      # get_all_items() returns all items, including the lid
+      # ignore the lid since it does not have a tracker
+      if not isinstance(well, Lid):
+        well.tracker.disable()
 
   def enable_volume_trackers(self) -> None:
     """Enable volume tracking for all wells in the plate."""
 
     for well in self.get_all_items():
-      well.tracker.enable()
+      # get_all_items() returns all items, including the lid
+      # ignore the lid since it does not have a tracker
+      if not isinstance(well, Lid):
+        well.tracker.enable()
```

Comment:
`get_all_items` should only return the items. If it doesn't (I can see how that would be the case, `get_item` calls `.children[identifier]`), there is an issue with ItemizedResource. In this case the root issue should be fixed.

Reply:
https://github.com/PyLabRobot/pylabrobot/pull/567